app/models/DecryptedSensitiveUserInformation.scala

```diff
@@ -14,17 +14,15 @@
  * limitations under the License.
  */
 
-package models.preferences
+package models
 
 import play.api.libs.json.{Json, OFormat}
 
-case class ContactPreference(
-  paperlessReference: String,
+case class DecryptedSensitiveUserInformation(
```

Comment:
As this is frontend code and will live in the public domain we do not want to draw attention to the fact we have dycrypted and sensitive user related data! Please rename this to something more generic that does not draw attention to this class.